apiary.apib

```diff
@@ -9833,20 +9833,22 @@ Schedule or immediately post content on  Social sites.
                             Instagram: 5000
                             LinkedIn: 3000
                             Twitter: 240
-        + socialSite: GOOGLEMYBUSINESS (string, required) - Social Sites. Possible values are
+        + socialSite: GOOGLEMYBUSINESS (string, required) - The social site where the post will be made. Possible values are
                                             INSTAGRAM,
```

Comment:
Correct the formatting there 
<img width="697" alt="format" src="https://github.com/user-attachments/assets/dd8b4843-ce7b-48ab-b507-ed8ac62cfdb5">

Reply:
Also there is oner more GOOGLEMYBUSINESS visible. at the bottom . Why is it required there